app/components/time_ago_component.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module Primer
+  # A basic wrapper for the "<time-ago>" element.
```

Comment:
I think we need something more descriptive here. we normally follow something like `Use TimeAgo to ...`